src/ios/BranchSDK.m

```diff
@@ -94,11 +94,11 @@ - (void)initSession:(CDVInvokedUrlCommand*)command
         isFromBranchLink = [[params objectForKey:@"+clicked_branch_link"] boolValue];
 
         if (!error) {
-            if (params != nil && [params count] > 0 && isFromBranchLink) {
+            if (params != nil && [params count] > 0) {
  
                 NSError *err;
                 NSData *jsonData = [NSJSONSerialization dataWithJSONObject:params options:0 error:&err];
-
+            if(isFromBranchLink){
```

Comment:
minor but please keep the style of the code around => put whitespace i.e. `if (isFromBranchLink) {`